cmd/kubeadm/app/cmd/init.go

```diff
@@ -224,10 +221,14 @@ func AddInitOtherFlags(flagSet *flag.FlagSet, cfgPath *string, skipPreFlight, sk
 		criSocket, "cri-socket", "/var/run/dockershim.sock",
 		`Specify the CRI socket to connect to.`,
 	)
+	flagSet.StringVar(
+		defaultUser, "default-user", *defaultUser,
+		"Sets the default, non-root user for this cluster. The cluster configuration file will be automatically copied to the home directory for this user.",
```

Comment:
This says non-root, but it seems like we don't check that anywhere.

Nit: `The cluster configuration file will be automatically copied to the home directory for this user.` => `The administrator's credentials to the cluster will be copied to this user's home directory`

Reply:
ack.

Reply:
question: does that affect the newly proposed naming for the flag: `--copy-config-for-user` vs `--copy-admin-credentials-for-user` vs `...` ?

Reply:
re: `non-root`: looks like i've forgot to add this check.